ec2/multi_node/README.md

```diff
@@ -0,0 +1,224 @@
+  * Configure a hostname with as a CNAME record through Route53
+  * Setup OpenShift via the [OpenShift Ansible BYO playbook](https://github.com/openshift/openshift-ansible/blob/master/playbooks/byo/config.yml)
+  * Install Service Catalog 
+  * Install Ansible Service Broker
+  * Configure cluster users with htpasswd authentication
+
+## Pre-Reqs
+  * Ansible needs to be installed so its source code is available to Python.
+    * Check to see if Ansible modules are available to Python
+      ```bash
+      $ python -c "import ansible;print(ansible.__version__)"
+      2.3.0.0
+      ```
+    * MacOS requires Ansible to be installed from `pip` and not `brew`
+      ```bash
+      $ python -c "import ansible;print(ansible.__version__)"
+      Traceback (most recent call last):
+      File "<string>", line 1, in <module>
+      ImportError: No module named ansible
+
+      brew uninstall ansible
+      pip install ansible
+
+      $ python -c "import ansible;print(ansible.__version__)"
+      2.3.0.0
+      ```
+  * Install python dependencies (This is needed for python2. Use pip3 if using python3)
+    * On Fedora and EL7 it is recommended that you use ansible in a python virtualenv.
+     * This is due to a couple reasons:
+       - boto rpms are not sufficiently new enough
+       - pip is not sudo safe on Fedora and EL7 
+     * To setup and activate a virtualenv do the following;
+     ```
+     sudo dnf install python-virtualenv #or EL7: sudo yum install python-virtualenv 
+     virtualenv /tmp/ansible
+     source /tmp/ansible/bin/activate
+     pip install ansible
+     ```
+   * Continue with the next step:
+    ```bash
+    $ pip install boto boto3 six
+    ```
+  * Configure a SSH Key in your AWS EC-2 account for the given region
+  * Create a hosted zone in Route53
+  * Set these environment variables:
+    ```bash
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    AWS_SSH_PRIV_KEY_PATH  - Path to your private ssh key to use for the ec2 instances
+    ```
+## Quick Start - Scenario Configuration & Execution
+The following will describe the *quick start* steps to configure various supported scenairos. All variables shown below needs to be in the `my_vars.yml` file, located in the [`config`](../../config) folder
+
+### OCP - Official CDN Repositories
+Deploying OCP in RHEL is the default configuration.  However, you MUST provide the RHN credentials and the pool ID with OpenShift Container Platform subscription
+```bash
+rhn_user: changeme
+rhn_passwd: changeme
+aos_pool_id: changeme
+```
+
+### OCP - Internal Mirror Repositories
+To deploy OCP in RHEL with the latest OCP packages from the '`aos-ansible`' Ops private repository, do the following
+```bash
+enable_ops_mirror_repo: true
+ops_mirror_dir: /git/aos-ansible
+```
+Note: The ops_mirror_dir folder MUST be valid and exist when configuring this scenario.
+
+
+### Origin
+To deploy Origin in Centos, set the following
+```bash
+deploy_origin: true
```

Comment:
There should be instructions here for how to configure the source for APBs.

Assuming that official OCP deployment would default to rhcc, yet origin would default to dockerhub and require a dockerhub org.